docs/tasks.md

```diff
@@ -146,6 +146,67 @@ or container images that you define:
   image in the Task, rather than requesting the sum of all of the container
   image's resource requests.
 
+#### Step Script
+
+To simplify executing scripts inside a container, a step can specify a `script`.
+If this field is present, the step cannot specify `command` or `args`.
+
+When specified, a `script` gets invoked as if it were the contents of a file in
```

Comment:
Should we validate that in the webhook ? (that it starts with a `shebang`)

Reply:
Maybe, sure. If we did, we'd probably only be able to validate that `strings.HasPrefix(script, "#!")` but that might be worth catching early.

tbh I'd love to be able to drop the requirement, but in my testing the entrypoint binary barfed without it, with `exec format error` and I couldn't figure out why.

Let me know if you'd like me to add the validation now while we need it.